addons/website/static/src/js/widgets/theme.js

```diff
@@ -373,6 +373,21 @@ var ThemeCustomizeDialog = Dialog.extend({
                         _processItems($item.children(), $element.find('.o_theme_customize_selection'), true);
                         break;
 
+                    case 'FONTSELECTION':
+                        var options = [];
+                        _.times(parseInt(self.style.getPropertyValue('--number-of-fonts')), function (font) {
+                            var option = $('<opt/>', {
+                                'data-widget': 'auto',
+                                'data-variable': $item.data('variable'),
+                                'data-value': font + 1,
+                                'data-font': font + 1,
+                            });
+                            options.push(option[0]);
+                        });
```

Comment:
Seeing the doc of `_.times`, you can do it that way:
```javascript
var options = _.times(..., function (font) {
    ...
    return $option[0];
});
```
But in fact, the `_processItems` method expects a jQuery objects for the items. So do:
```javascript
var $options = $();
_.times(..., function (font) {
    ...
   $options = $options.add($option);
});
```